devolutions-gateway/src/config.rs

```diff
@@ -931,24 +935,6 @@ impl Config {
             }
         }
 
-        // NOTE: we allow configs to specify "http" or "https" scheme if it's clearer,
-        // but this is ultimately identical to "ws" and "wss" respectively.
-
-        let has_at_least_one_http_listener = listeners
```

Comment:
Why did you remove that ? We don't want to panic early if there is no listener defined ?

Reply:
This was duplicate logic, the very same check is already applied later in the `validate` function (and also includes listeners provided through CLI).

Reply:
Excellent